rpcserverhelp.go

```diff
@@ -99,6 +99,14 @@ var helpDescsEnUS = map[string]string{
 	"decodescript--synopsis": "Returns a JSON object with information about the provided hex-encoded script.",
 	"decodescript-hexscript": "Hex-encoded script",
 
+	// GenerateCmd help
+	"generate--synopsis": "Generates a set number of blocks (simnet or regtest only) and returns a JSON\n" +
+		" array of their hashes.",
+	"generate-numblocks": "Number of blocks to generate",
+
+	// GenerateResult help
```

Comment:
Super Nit: This comment can go away since the type has been removed.   Once that goes away, so should the blank line above it.